website/content/docs/platform/servicenow/installation.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: Vault Credential Resolver Installation
+description: Installation steps for the ServiceNow Vault Credential Resolver.
+---
+
+# Installing the Vault Credential Resolver
+
+## Prerequisites
+
+* ServiceNow version Quebec+ (untested on previous versions)
+* MID server version Quebec+ (untested on previous versions)
+* Discovery and external credential plugins activated on ServiceNow
+* Working Vault deployment accessible from the MID server
+
+## Installing Vault Agent
+
+* Select your desired auth method from Agent's [supported auth methods](/docs/agent/autoauth/methods)
+  and set it up in Vault
+  * For example, to set up AppRole auth and a role called `role1` with the `demo` policy attached:
+
+    ```bash
+    vault auth enable approle
+    vault write auth/approle/role/role1 bind_secret_id=true token_policies=demo
+    ```
+
+  * To get the values required for files `/vault/config/roleID` and
+    `/vault/config/secretID` referenced in the config below, you could then run:
+
+    ```bash
+    roleID=$(vault read -format json auth/approle/role/role1/role-id | jq -r '.data.role_id')
+    secretID=$(vault write -format json -f auth/approle/role/role1/secret-id | jq -r '.data.secret_id')
+    ```
+
+* Create an `agent.hcl` config file. Your exact configuration may vary, but you
+  must set `cache.use_auto_auth_token = true`, and the `listener`, `vault` and
+  `auto_auth` blocks are also required to set up a working Agent, e.g.:
+
+  ```
+  listener "tcp" {
+    address = "127.0.0.1:8200"
+    tls_disable = true
+  }
+
+  cache {
+    use_auto_auth_token = true
+  }
+
+  vault {
+    address = "http://vault.example.com:8200"
+  }
+
+  auto_auth {
+      method {
+          type = "approle"
+          config = {
+              role_id_file_path = "/vault/config/roleID"
+              secret_id_file_path = "/vault/config/secretID"
+          }
+      }
+  }
+  ```
+
+  * Install Agent as service running `vault agent -config=/path/to/agent.hcl`
+    * Documentation for Windows service installation [here](/docs/agent/winsvc)
+
+## Uploading JAR file to MID server
+
+* Download the Vault Credential Resolver JAR file from **TBD**
```

Comment:
I guess we'll update this when we have a source the JAR.

Reply:
For the initial release, it's only going to be available at releases.hashicorp.com. I've added that link now, though it will be a 404 until we actually hit the publish button